### ISSUE TITLE
Does Zeropadding requires a memory in Keras?

### ISSUE THREAD

**Issue by sekti92** [keras-team/keras#12204]
Hello guys, i want to know does zero padding requires a memory in neural network?
I red **VGGNet in detail** section in http://cs231n.github.io/convolutional-networks/
But it doesn't contains zero padding.
Anybody?
